Quintor/src/main/java/com/example/quintor/ApiService.java

```diff
@@ -0,0 +1,80 @@
+package com.example.quintor;
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.ProtocolException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+
+public class ApiService {
+
+    private URL url;
+
+    public ApiService(){
+        setUrl("http://localhost:8080/MT940toJSON");
+    }
+
+    public String getRequest() throws Exception {
+        HttpURLConnection connection = setConnection("get");
+
+        BufferedReader in = new BufferedReader(new InputStreamReader(connection.getInputStream()));
+        String inputLine;
+        StringBuffer response = new StringBuffer();
+
+        while ((inputLine = in.readLine()) != null) {
+            response.append(inputLine);
+        }
+        in.close();
+
+        return response.toString();
```

Comment:
Deze regels zijn precies hetzelfde als regel 40 tm 49 is er een manier waardoor dit in een aparte methode kan worden gezet?